### PR TITLE
Added 6 new files with resolved tasks for lesson 4

### PR DESCRIPTION
**Pull request by h96ven**

### CODE REVIEW

File: liubarskyi/HW4/4_1.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	century = year // 100 + 1
+
+
+if century > 0:
+	print(century, 'century AD!')
+
+elif century < 0:
+	print(-century, 'century BC!')
+
+print('-----------------------------------')
```
Comment (katemalash) [inline liubarskyi/HW4/4_1.py:31]: So good decision)

File: liubarskyi/HW4/4_2.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	triangle_area = (p * (p - a) * (p - b) * (p - c)) ** 0.5
+	print(f"The square area of the triangle: {triangle_area}")
+
+elif figure == 'circle' or figure == 'c':
+	r = int(input('Specify the radius of the circle '))
+	circle_area = pi * r**2
+	print(f"The square area of the circle: {circle_area}")
+
+else:
+	print("Error! You didn't choose it right. Please, try again. ")
```
Comment (katemalash) [inline liubarskyi/HW4/4_2.py:28]: Nice!

File: liubarskyi/HW4/4_3.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+digit_counter = 0
+
+if our_number != 0:
+	while our_number > 0:
+		digit_counter += 1
+		our_number = our_number // 10
+	print(f"It is a {neg_pos} {digit_counter}-digit number!")
+elif our_number == 0:
+	print('The number is zero!')
+
```
Comment (katemalash) [inline liubarskyi/HW4/4_3.py:20]: Good Decision but too many empty lines

File: liubarskyi/HW4/4_4.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+side of the square hall! ')
+
+
+if user_radius <= radius_small:
+	print("Yes! It's possible! It can be successfully done! ")
+elif user_radius > radius_small:
+	print("No! Not possible! The desired radius of the circle is too big! ")
+elif user_radius <= 0:
+	print("The desired radius cannot be zero or negative! ")
+
```
Comment (katemalash) [inline liubarskyi/HW4/4_4.py:36]: Good Decision but too many empty lines

File: liubarskyi/HW4/4_5.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                
+note_quantity = [0, 0, 0, 0]
+     
+print ("Quantity of bank notes: ")
+     
+for i, j in zip(bank_notes, note_quantity):
+    if amount >= i:
+        j = amount // i
+        amount = amount - j * i
+        print (i,":", j)
```
Comment (katemalash) [inline liubarskyi/HW4/4_5.py:13]: This is good and correct way

File: liubarskyi/HW4/4_5.py [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+# 	note_1_qntt = amount 
+# 	amount = amount - note_1_qntt
+
+# elif amount <= 0:
+# 	print('Amount of money cannot be zero or negative! ')
+	
+# print('-----------------------------------------------------')
+
+# print(f"Note 200: {note_200_qntt}, note 100: {note_100_qntt}, \
+# note 10: {note_10_qntt}, note 1: {note_1_qntt}")
```
Comment (katemalash) [inline liubarskyi/HW4/4_5.py:86]: ... but this one also works

File: liubarskyi/HW4/4_6.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+	print('------------------------------------------------------------')
+
+	if user_place % 2 == 1:
+		print('Your place is a bottom place! ')
+		
+	elif user_place % 2 == 0:
+		print('Your place is a top place! ')
+
+elif user_place > 54:
+	print('There is no such place! Specify a number between 1 and 54! ')
```
Comment (katemalash) [inline liubarskyi/HW4/4_6.py:31]: Good

**Review by katemalash — APPROVED** [review]
Good Job. Please merge this pull-request

File: liubarskyi/HW4/4_1.py (hunk 1 above)
Comment (h96ven) [reply liubarskyi/HW4/4_1.py:31]: Thanks

File: liubarskyi/HW4/4_2.py (hunk 2 above)
Comment (h96ven) [reply liubarskyi/HW4/4_2.py:28]: Thank you!

File: liubarskyi/HW4/4_3.py (hunk 3 above)
Comment (h96ven) [reply liubarskyi/HW4/4_3.py:20]: Thanks, okay

File: liubarskyi/HW4/4_4.py (hunk 4 above)
Comment (h96ven) [reply liubarskyi/HW4/4_4.py:36]: Thanks, okay

File: liubarskyi/HW4/4_5.py (hunk 5 above)
Comment (h96ven) [reply liubarskyi/HW4/4_5.py:13]: Thanks

File: liubarskyi/HW4/4_6.py (hunk 7 above)
Comment (h96ven) [reply liubarskyi/HW4/4_6.py:31]: Thank you

File: liubarskyi/HW4/4_5.py (hunk 6 above)
Comment (h96ven) [reply liubarskyi/HW4/4_5.py:86]: Yeah

**Pull request merged**